addons/stock/models/stock_rule.py

```diff
@@ -297,18 +300,21 @@ def _get_custom_move_fields(self):
         """
         return []
 
+    def _get_group_id(self, current_group_id):
```

Comment:
```suggestion
    def _get_group_id(self, current_group):
```
Just to make things clearer, as the method ends up with `_id` and returns an id, while the parameter ends up with `_id` but is a recordset.

Reply:
done